ChangeLog

```diff
@@ -1,16 +1,23 @@
 
 ### 3.8.0rc2dev <- NOTE: the release version number will be 3.8.0 ###
 
+- The term "Central server" has been replaced with "Directory server" (#1407, #1715, #1629).
+  Note that the program will still accept the --centralserver option for backward
+  compatibility with existing system startup scripts.
```

Comment:
I just assumed "backward  compatibility with existing" meant the same thing for us as "deprecated", but does adding that word here mean we actually intend to remove that compatibility at some point?

Reply:
I guess
> will still accept

could be read as
> will always still accept

and there's no guarantee of that.  At some stage the command line handling may get rewritten and it could get dropped then...

Reply:
Added a hint similar to what @ann0see suggested.